src/CreateLesson/CreateLesson.test.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter } from 'react-router-dom';
+import { render, fireEvent, getByLabelText, waitFor } from '@testing-library/react';
+import CreateLesson from './CreateLesson';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux'
+import { rootReducer } from '../reducers';
+import { addLesson } from '../mockData/mockData'
+
+const store = createStore(rootReducer, {
```

Comment:
Great work with the integration tests!

Reply:
Nice work.  We should consider sad paths for future tests but i think a lot of them will be on the child components